knowledge_repo/app/models.py

```diff
@@ -350,8 +350,18 @@ def groups(self, groups):
 
         # create an implicit group, group_post.id, to add
         # single users to
-        group = Group(name="group_" + self.id)
+        # we don't have self.id here necessarily, so query the Post table
+        # for the path
+        post_id = (db_session.query(Post)
+                             .filter(Post.path == self.path)
+                             .first()).id
+        group = Group(name="group_" + str(post_id))
```

Comment:
I think not, I think it's useful to always just have a default group available for people to add to. But I'll definitely rename it so it doesn't conflict with any user specified group names